lib/routes/gov/yunnan/ynrsjzp.ts

```diff
@@ -0,0 +1,71 @@
+import { Route, Data } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import type { Context } from 'hono';
+
+export const route: Route = {
+    path: '/yunnan/ynrsjzp/:page?',
+    url: 'hrss.yn.gov.cn/',
+    categories: ['government'],
+    example: '/gov/yunnan/ynrsjzp',
+    parameters: {
+        page: '需要订阅的页数，默认为1',
+    },
+    radar: [
+        {
+            source: ['hrss.yn.gov.cn/'],
+        },
+    ],
+    name: '云南省人力资源和社会保障政务门户-招考招聘',
+    maintainers: ['MajexH'],
```

Comment:
```suggestion
    maintainers: ['shenleg'],
```
This should be your GitHub ID